packages/tocco-ui/src/Panel/Panel.js

```diff
@@ -65,7 +74,11 @@ Panel.propTypes = {
   /**
    * Boolean to control if body can be opened or closed. Default value is 'true'.
    */
-  isToggleable: PropTypes.bool
+  isToggleable: PropTypes.bool,
+  /**
+   * Function gets passed from PanelGroup to ensure that only one Panel is open at one time.
```

Comment:
documentation: The Panel doesn't care if "only one Panel is open at one time". onToggle is simply a function that is called whenever the Panel is closed or opened.